examples/resources/plugins/teleport-gitlab.toml

```diff
@@ -1,24 +1,24 @@
 # example teleport-gitlab configuration TOML file
 [teleport]
-auth-server = "example.com:3025"  # Auth GRPC API address
-client-key = "/var/lib/teleport/plugins/gitlab/auth.key" # Teleport GRPC client secret key
-client-crt = "/var/lib/teleport/plugins/gitlab/auth.crt" # Teleport GRPC client certificate
-root-cas = "/var/lib/teleport/plugins/gitlab/auth.cas"   # Teleport cluster CA certs
+auth_server = "example.com:3025"                         # Teleport Auth Server GRPC API address
+client_key = "/var/lib/teleport/plugins/gitlab/auth.key" # Teleport GRPC API client secret key
+client_crt = "/var/lib/teleport/plugins/gitlab/auth.crt" # Teleport GRPC client certificate
+root_cas = "/var/lib/teleport/plugins/gitlab/auth.cas"   # Teleport cluster CA certs
 
 [db]
 path = "/var/lib/teleport/plugins/gitlab/database" # Path to the database file
 
 [gitlab]
 url = ""                                   # Leave empty if you are using cloud
 token = "token"                            # GitLab API Token
-project-id = "1812345"                     # GitLab Project ID
-webhook-secret = "your webhook passphrase" # A secret used to encrypt data we use in webhooks. Basically anything you'd like.
+project_id = "1812345"                     # GitLab Project ID
+webhook_secret = "your webhook passphrase" # A secret used to encrypt data we use in webhooks. Basically anything you'd like.
 
 [http]
-listen = ":8081"                                                  # Webhook listener
-base-url = "https://your-server.example.com/teleport-gitlab"      # URL by which bot is accessible — will be used in Gitlab webhook settings
-# https-key-file = "/var/lib/teleport/plugins/gitlab/server.key"  # TLS private key
-# https-cert-file = "/var/lib/teleport/plugins/gitlab/server.crt" # TLS certificate
+public_addr = "example.com" # URL on which webhook server is accessible externally, e.g. [https://]teleport-gitlab.example.com
+# listen_addr = ":8081" # Network address in format [addr]:port on which webhook server listens, e.g. 0.0.0.0:443
```

Comment:
Port at the end should match the port at the beginning

i.e. `# Network address in format [addr]:port on which webhook server listens, e.g. 0.0.0.0:8081`